.travis.yml

```diff
@@ -1,4 +1,7 @@
 ---
+AWS_ACCESS_KEY_ID: b1ugrISCMawg4LDOEnj91Eu3eb8EnxwOD63Y9owfxdaclmp/K4wOUdRXLE8Gmv0FXbzPJXbMld6a4wx0qfq95pQMy22gut0o4e9IAxPguhIeF9wFzZdw2nQFTEV2+T4Y52QIhPXnV403bt4hZZbIzfniwVRsdHoac60t9NoyBOI=
+AWS_ACCESS_KEY_SECRET: X1tocl6D1nwzVtUjlR9n/PbtkhtgyGjmK0qUyL5DeTX8bY6pcIMZU3g5S1yoseMzV3DnjyoRrO1vZibFFRTPPa2iDVtMAjuVyKeQVhkMl++hslcF4QsO/qlFJJg2B2twlanKQDmiRG14dM032wyQcbHLQKVFW6yF6XoW4ZTUYBk=
+
```

Comment:
These types of keys are typically encrypted, which may be the case here (they are not generally this long when unencrypted).

@joostdevries can confirm (or fix :dancer:)...